components/features/console/SelectSchool.tsx

```diff
@@ -28,9 +28,9 @@ export default function SelectSchool({ setSchoolAction }: Props) {
   if (isLoading) {
     return <SpinnerIcon className="mx-auto animate-spin" size={25} />;
   }
-  if (!schoolsList) return;
+  if (!schools) return;
```

Comment:
What does that do? also returning undefined is not recommended.
```suggestion
  if (!schools) return <></>;
```